MovieDB/MovieDB/Scenes/Movie Detail/MovieDetailCoordinator.swift

```diff
@@ -0,0 +1,30 @@
+import UIKit
+
+protocol MovieDetailCoordinatorType {
+    func finish()
```

Comment:
As comment before, the start() and finish() should be declared on Coordinator protocol, because they is main functions of it. And MovieDetailCoordinatorType have to inherit the Coordinator, not alone.